src/frontend/src/icp/components/send/IcSendTokenWizard.svelte

```diff
@@ -190,6 +191,15 @@
 		store: initEthereumFeeStore()
 	});
 
+	const { store: ethereumFeeStore } = getContext<EthereumFeeContext>(ETHEREUM_FEE_CONTEXT_KEY);
+
+	// In case we are converting ckERC20 to ERC20, we need to include ckETH related fees in the transaction.
+	let ckErc20ToErc20MaxCkEthFees: bigint | undefined = undefined;
```

Comment:
Is there any particular reason why you are assigning a state value? We do not need this for the UI right?

If the above is correct and given the fact that the transfer is executed in an imperative way, I would maybe also assign `ckErc20ToErc20MaxCkEthFees` imperatively as we already do for example with `$tokenAsIcToken`. WDYT?

```
// In case....
const ckErc20ToErc20MaxCkEthFees = isNetworkIdETH(networkId) && $tokenCkErc20Ledger ? $ethereumFeeStore?.maxTransactionFee : undefined;

const params: IcTransferParams = {
				to: destination,
				amount: parseToken({
					value: `${amount}`,
					unitName: $token.decimals
				}),
				identity: $authStore.identity,
				progress: (step: ProgressStepsSendIc) => (sendProgressStep = step),
				ckErc20ToErc20MaxCkEthFees
			};

await sendIc({
				...params,
				token: $tokenAsIcToken,
				targetNetworkId: networkId
			});
```

Reply:
good point! changed